lib/ProductOpener/APIProductServices.pm

```diff
@@ -51,6 +51,7 @@ Currently implemented services:
 - extend_ingredients : extend the ingredients object with additional information
 - estimate_ingredients_percent : compute percent_min, percent_max, percent_estimate for each ingredient in the ingredients object
 - analyze_ingredients : analyze the ingredients object and return a summary object
+- estimate_environmental_cost_ingredients : estimate the environmental cost of a given product (see Ecobalyse)
```

Comment:
I think the service should be estimate_environmental_impact, without speciffying ingredients: at first we will look only at ingredients, but maybe in the next iteration, we will also add the packaging data, and maybe some processing implied by the category (e.g. frozen vegetables: freezing).

cost -> I think it may be more generic to use "impact" instead of "cost"

So we could create a separate EnvironmentalImpact.pm module.